lib/src/main/java/com/cepheuen/elegantnumberbutton/view/ElegantNumberButton.java

```diff
@@ -113,9 +114,85 @@ public void onClick(View mView) {
                 setNumber(String.valueOf(num + 1), true);
             }
         });
+        
+        subtractBtn.setOnLongClickListener(new OnLongClickListener() {
+            @Override
+            public boolean onLongClick(View v) {
+                dec_pressed = true;
+                touchUpdateHandler.post( new RptUpdater() );
+                return true;
+            }
+        });
+        addBtn.setOnLongClickListener(new OnLongClickListener() {
+            @Override
+            public boolean onLongClick(View v) {
+                inc_pressed = true;
+                touchUpdateHandler.post( new RptUpdater() );
+                return true;
+            }
+        });
+        subtractBtn.setOnTouchListener(TouchListener);
+        addBtn.setOnTouchListener(TouchListener);
         a.recycle();
     }
 
+    
+    //################### Auto Increment on Long Press ########################
+    private boolean inc_pressed = false;
+    private boolean dec_pressed = false;
+    private int _DELAY = 100;
+    private Handler touchUpdateHandler = new Handler();
+    //private boolean mAutoIncrement = false;
+    //private boolean mAutoDecrement = false;
+    //public int mValue;
+    class RptUpdater implements Runnable {
```

Comment:
It's not a good idea to declare a sub-class within a class to perform an operation. Try to separate the class out and try to pass the variables instead of using global variables